resources/definitions/desktop.ini

```diff
@@ -0,0 +1,5 @@
+[.ShellClassInfo]
```

Comment:
This seems to be a file from another tool, not needed by Cura. Please remove or ignore it from Git.

Reply:
There's more of these, please remove all.

Reply:
Fixed. Forgot i had hidden files off on this computer.